ERROR:root:You messed up. Here is the error:
'nctId'
Traceback (most recent call last):
  File "C:\Users\wasab\Documents\Moffit Internship\Chat GPT Prompts\Code\main.py", line 41, in process_trials
    trialGetter.saveTrialToFile(trial_json, folder, suffix="_Processed")
  File "C:\Users\wasab\Documents\Moffit Internship\Chat GPT Prompts\Code\newRawDataController.py", line 35, in saveTrialToFile
    fileName = os.path.join(folder, f"{trial['nctId']}_{suffix}.json")
                                       ~~~~~^^^^^^^^^
KeyError: 'nctId'

ERROR:root:You messed up. Here is the error:
'nctId'
Traceback (most recent call last):
  File "C:\Users\wasab\Documents\Moffit Internship\Chat GPT Prompts\Code\main.py", line 41, in process_trials
    trialGetter.saveTrialToFile(trial_json, folder, suffix="_Processed")
  File "C:\Users\wasab\Documents\Moffit Internship\Chat GPT Prompts\Code\newRawDataController.py", line 36, in saveTrialToFile
    fileName = os.path.join(folder, f"{trial['nctId']}_{suffix}.json")
                                       ~~~~~^^^^^^^^^
KeyError: 'nctId'

